examples/demo-apps/android/LlamaDemo/android-llm-device-farm-test-spec.yml

```diff
@@ -73,16 +73,29 @@ phases:
           fi
         fi;
 
-      # Run the new generic benchmark activity
+      # Run the new generic benchmark activity https://developer.android.com/tools/adb#am
       - echo "Run LLM benchmark"
       - |
-        adb -s $DEVICEFARM_DEVICE_UDID shell am start -n com.example.executorchllamademo/.LlmBenchmarkRunner \
+        adb -s $DEVICEFARM_DEVICE_UDID shell am start -W -n com.example.executorchllamademo/.LlmBenchmarkRunner \
         --es "model_dir" "/data/local/tmp/llama" \
         --es "tokenizer_path" "/data/local/tmp/llama/tokenizer.bin"
 
   post_test:
     commands:
-      - adb -s $DEVICEFARM_DEVICE_UDID shell "ls -la /data/local/tmp/llama/"
+      - echo "Wait for the results"
+      - |
+        # TODO (huydhn): Polling like this looks brittle, figure out if there is a better way to wait
```

Comment:
Curious what could be brittle with this approach?

Reply:
I just feel that there might be a better approach out there, so put a TODO here to remind myself for now